po/de.po

```diff
@@ -318,53 +304,56 @@ msgstr "Heimordner für %ls"
 #: src/complete.cpp:63
 #, c-format
 msgid "Variable: %ls"
-msgstr ""
+msgstr "Variable: %ls"
 
 #: src/complete.cpp:66
 #, c-format
 msgid "Abbreviation: %ls"
-msgstr ""
+msgstr "Abkürzung: %ls"
 
 #: src/complete.cpp:1514
 msgid "completion reached maximum recursion depth, possible cycle?"
 msgstr ""
+"Vervollständigung hat die maximale Rekursionstiefe erreicht, möglicher Kreis?"
 
 #: src/env.cpp:1340
 msgid ""
 "Could not determine current working directory. Is your locale set correctly?"
 msgstr ""
+"Das aktuelle Arbeitsverzeichnis konnte nicht bestimmt werden. Ist die "
+"Lokalisierung korrekt eingestellt?"
 
 #: src/env_dispatch.cpp:471
 #, c-format
 msgid "Using fallback terminal type '%s'."
-msgstr ""
+msgstr "Nutze Rückfallterminaltyp '%s'."
 
 #: src/env_dispatch.cpp:474
 #, c-format
 msgid "Could not set up terminal using the fallback terminal type '%s'."
-msgstr ""
+msgstr "Konnte mit dem Rückfallterminaltyp '%s' kein Terminal einrichten."
 
 #: src/env_dispatch.cpp:575
 msgid "Could not set up terminal."
-msgstr ""
+msgstr "Konnte das Terminal nicht einrichten"
 
 #: src/env_dispatch.cpp:577
 msgid "TERM environment variable not set."
-msgstr ""
+msgstr "TERM Umgebungsvariable nicht gesetzt."
 
 #: src/env_dispatch.cpp:579
 #, c-format
 msgid "TERM environment variable set to '%ls'."
-msgstr ""
+msgstr "TERM Umgebungsvariable auf '%ls' gesetzt."
 
 #: src/env_dispatch.cpp:581
 msgid "Check that this terminal type is supported on this system."
-msgstr ""
+msgstr "Überprüfe, ob dieser Terminaltyp auf diesem System unterstützt wird."
 
 #: src/env_universal_common.cpp:425
 #, c-format
 msgid "Unable to write to universal variables file '%ls': %s"
-msgstr ""
+msgstr "Die globale Variablen-Datei '%ls' kann nicht geschrieben werden: %s"
```

Comment:
"global" is already a term for "global variables", which isn't the same as universal variables. So... "Universalvariablendatei"? "Datei für universelle Variablen"?